Felienne/hedy
# [BUG FIX] Fixes styling of reset page

**Description**
The reset page missed out on our re-styling a while back. This might be due to it nog being accessible locally as it depends on a token to be there otherwise you receive an error page. We fix some margins and styling so the page looks nice and is responsive.

**Fixes**
This PR fixes #3173.

**How to test**
Merge to alpha, request a recover password link. Press it and verify that the UI now looks as expected.
